pkgs/applications/blockchains/monero-cli/default.nix

```diff
@@ -53,7 +52,11 @@ stdenv.mkDerivation rec {
     hash = "sha256-nDiFJjhsISYM8kTgJUaPYL44iyccnz5+Pd5beBh+lsM=";
   };
 
-  patches = [ ./use-system-libraries.patch ];
+  patches = [
+    ./use-system-libraries.patch
+    # https://github.com/monero-project/monero/pull/9462
```

Comment:
Can you please use `fetchpatch` to download the [merged commit](https://github.com/monero-project/monero/commit/83dd5152e6d115426afbb57a94a832ec91b58a46.patch)?

Reply:
Sure!